tlatools/org.lamport.tlatools/src/tla2sany/parser/ParseException.java

```diff
@@ -12,25 +13,25 @@
  */
 public class ParseException extends Exception {
 
+  /**
+   * The version identifier for this Serializable class.
+   * Increment only if the <i>serialized</i> form of the
+   * class changes.
+   */
+  private static final long serialVersionUID = 1L;
```

Comment:
I don't know, probably nowhere. But this is generated code so is not something we have control over.

Reply:
We used to have control over it when the generated code was checked into the repo.  Perhaps, the decision to always generate the code has to be revisited.

Reply:
I would strongly disagree with that idea since the ability to modify the parser source is hard-fought and paramount. I don't really see the problem with this line, the Java linter will give you warnings about it if it's missing for any custom exception class you define and it is a good idea for generated code to not contain warnings.

Reply:
Hard-fought yes, but why is it paramount?

Reply:
There are still a lot of parser bugs, for example, and we can expect to find more in the future. We also might want to change the way the TLA⁺ parser works. Both of these require changing the JavaCC grammar. I view re-synchronizing the JavaCC grammar with the source files one the premier improvements to the codebase to come out of the Unicode work.